arduino/walt/walt.ino

```diff
@@ -219,6 +219,8 @@ void process_command(char cmd) {
       init_vars();
       send_ack(CMD_RESET);
     } else if (cmd == CMD_GSHOCK) {
+      Serial.print(flip_case(cmd));
+      Serial.print(" ");
```

Comment:
I would say, let's still use sendReceive directly for now in readLastShockTime to merge the rest of this in, and deal with the protocol breaking stuff separately, this will also give a better documented change of protocol with a short git commit.